DWAPlannerROS:

    # Robot Config
    acc_lim_x: 0.5
    acc_lim_y: 0.5
    acc_lim_theta: 0.5
    
    max_vel_x: 0.3
    min_vel_x: -0.1
    max_vel_y: 0.1
    min_vel_y: -0.1
    max_vel_trans: 0.3
    min_vel_trans: 0.05
    max_vel_theta: 0.5
    min_vel_theta: 0.0
    
    # Global Tolerance
            
    yaw_goal_tolerance: 0.05
    xy_goal_tolerance: 0.05
    latch_xy_goal_tolerance: false
    
    # Forward Simulation Params
    
    sim_time: 2.0
    sim_granularity: 0.02
    vx_samples: 20
    vy_samples: 10
    vth_samples: 20
    controller_frequency: 20.0
    
    # Trajectory Scoring
    
    path_distance_bias: 32.0
    goal_distance_bias: 20.0
    occdist_scale: 0.02
    forward_point_distance: 0.325
    stop_time_buffer: 0.2
    scaling_speed: 0.25
    max_scaling_factor: 0.2
    
    # Oscillation Prevention
    
    oscillation_reset_dist: 0.05
    
    # Global Plan Parameters
    publish_traj_pc : true
    publish_cost_grid_pc: true 
